src/main/java/com/faforever/client/theme/UiService.java

```diff
@@ -472,4 +472,11 @@ public JFXDialog showInDialog(StackPane parent, Node content, String title) {
     dialog.show(parent);
     return dialog;
   }
+
+  @SneakyThrows
+  public boolean doesThemeNeedRestart(Theme theme) {
+    try (Stream<Path> stream = Files.list(getThemeDirectory(theme))) {
+      return stream.anyMatch(path -> Files.isRegularFile(path) && !path.endsWith(".css"));
```

Comment:
This will always return the theme.properties file

Reply:
What are u talking about? This is a check on wether a theme conatins a file that is not an css file, because if so we need a restart to fully load the theme.

Reply:
oh yeah i get u now. Hmm yes so I guess I got to include this there as well